icons.js

```diff
@@ -23,8 +23,9 @@ var getIcon = function getIcon(playerAddress){
 
 	let firstDot = addressWithoutMPRIS.indexOf(".");
 
-	let suspectAppName =
-		addressWithoutMPRIS.substring(0,firstDot);
+	let suspectAppName = addressWithoutMPRIS
+	if (firstDot > 1)
```

Comment:
I don't get this

Reply:
on my system at least, spotify and rhythmbox identify as `org.mpris.MediaPlayer2.spotify` and `org.mpris.MediaPlayer2.rhythmbox` respectively. firstDot is equal to -1 in those cases and results in a blank `suspectName`:
![image](https://user-images.githubusercontent.com/56710655/204682056-71fca723-35e2-4a71-b34b-eac1b56003fd.png)
the if check allows to avoid this:
![image](https://user-images.githubusercontent.com/56710655/204682195-b4440ef0-a48f-44eb-9ba8-27be9a6589e4.png)